phoneApp/js/user.js

```diff
@@ -0,0 +1,70 @@
+class User {
+  	constructor(globalState){
+    this.state = globalState; //стал равен this.state-у со страницы App.js
```

Comment:
something went wrong with the formatting. Try to install prettier to improve your code-base formatting.